modules/service/data-service.js

```diff
@@ -16,7 +16,11 @@ class DataService {
 
     initialize() {
```

Comment:
I was not sure, whether you prefer the url being derived before constructor call or within.
If you want to have it differently just comment out the url: and outcomment the host+port+tls variable


It can be done either by
- Deriving the url inside the constructor.
- Handing over the url to the constructor